src/components/L2/Dropdowns/FeeDropdown.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+
+import Dropdown from './Dropdown';
+
+import './FeeDropdown.scss';
+
+interface Fee {
+  amount: number;
+  display: string;
+}
+
+const FeeDropdown = () => {
+  const [open, setOpen] = useState<boolean>(false);
+  const [fee, setFee] = useState<Fee>({
+    amount: 0.0002,
+    display: '0.0002 ETH (15 min)',
+  });
+
+  const fees = [
+    {
+      amount: 0.0004,
+      display: '0.0004 ETH (10 min)',
+    },
+    {
+      amount: 0.0002,
+      display: '0.0002 ETH (15 min)',
+    },
+    {
+      amount: 0.0002,
+      display: '0.0001 ETH (1 hour)',
+    },
+  ];
```

Comment:
Are these the values we will be using in production? Or is there an API where we can load updated gas prices?

Reply:
Right now I'm not using this component. I can remove these but generally yes, they should be coming from an API